state/state.go

```diff
@@ -932,27 +932,36 @@ func (s *State) LoadDBState(dbheight uint32) (interfaces.IMsg, error) {
 
 	nextABlock, err := s.DB.FetchABlockByHeight(dbheight + 1)
 	if err != nil || nextABlock == nil {
-		return nil, err
-	}
-	abEntries := nextABlock.GetABEntries()
-	for _, adminEntry := range abEntries {
-		data, err := adminEntry.MarshalBinary()
-		if err != nil {
-			return nil, err
+		if dbheight < s.ProcessLists.DBHeightBase || int(dbheight) > int(s.ProcessLists.DBHeightBase)+len(s.ProcessLists.Lists) {
```

Comment:
This test isn't necessary.  You can just get the process list, and if it is nil, then you don't have one.